data-modules/output/asn/AuthenticationFramework.asn

```diff
@@ -19,9 +19,9 @@ Validity ::= CHOICE {
 		
 ValidityValue ::= SEQUENCE {
     notBefore       GeneralizedTime,
-    notBeforeInt    INTEGER OPTIONAL, -- NotBefore time since epoch in milliseconds
+    notBeforeInt    INTEGER OPTIONAL, -- NotBefore time since epoch in seconds
     notAfter        GeneralizedTime,
-    notAfterInt     INTEGER OPTIONAL -- NotAfter time since epoch in milliseconds
+    notAfterInt     INTEGER OPTIONAL -- NotAfter time since epoch in seconds
```

Comment:
Yes, they are definitely needed for something running on the smart contract. However, optional for the any web2 usage. Because if we make them all required then the schema is not backward compatible with old attestations (furthermore GeneralizedTime becomes totally superfluous and should just be removed).
However, as we talked about loosing backward compatibility is not really an issue since we have only issued 1 hour valid attestations for awhile, and I think it is fair to deprecated those few "unlimited time" old attestations we issued. 
So ideally I would just want to remove GeneralizedTime and deprecate the initial attestation, but I think that should be cleared with Weiwu. This would also require many code changes.

Reply:
The logic in all the code bases (including smart contract), is that notAfterInt is optional but required to work on smart contracts, and when new attestations are issued notAfterInt and notBeforeInt _will always_ be set unless explicitly requested to not be included.

Reply:
yes, correct, but do we use option to force old legasy format somewhere? As I know - we dont use it and always add int values.
even more - libs updated to always read Int and GeneralizedTime both(if exists) and compare them, so in real usage we always use new format only @jot2re

Reply:
Yes, true. I do think the best would simply be to get rid of the GeneralizedTime field. But I don't think we should do that before we get a good and integrated CI/CD setup.
I can change the backend code to _always_ enforce that notBeforeInt and notAfterInt are included, but in that case we should make a separate issue, since this is not related to NPM packaging ;)

Reply:
ok, lets leave it for separate issue

Reply:
Added a new issue #249